### ISSUE TITLE
Barrierefreiheit - not full accessable

### ISSUE THREAD

**Issue by neffets** [heiseonline/shariff#90]
In US you have WCAG and article-508, in Germany there exist BITV.

While testing for BITV we discovered two possible problems:

1) the color-contrast of the share-buttons and counters in theme "white" are not high enough.

2) BITV Prüfschritt: 1.3.2b http://testen.bitvtest.de/index.php?a=di&iid=37&s=n
requires, that the page is good readable without CSS enabled too.
Problem: without CSS the text of the share-Button and the Counter of the share button have no space between. So its not readable.
Possible solution could be to provide in <span class="share-count"> 4</span> allways a space before the number, or to in case of counter-show: to add first a space and then the share-count-span.

**Comment by neffets** [comment]
In Version v1.11.0 I suggest in the append for share_count a simple further SPACE before the span

```javascript
--- a/src/js/shariff.js
+++ b/src/js/shariff.js
@@ -155,7 +155,7 @@ Shariff.prototype = {
             if(value >= 1000) {
                 value = Math.round(value / 1000) + 'k';
             }
-            $(self.element).find('.' + key + ' a').append('<span class="share_count">' + value);
+            $(self.element).find('.' + key + ' a').append(' <span class="share_count">' + value);
         });
     },
 
```

**Comment by compeak** [comment]
> Problem: without CSS the text of the share-Button and the Counter of the share button have no space between. So its not readable.

I added a ```&nbsp;``` in 897a035f09fd928b2854cabf23c586317f025c27.

> the color-contrast of the share-buttons and counters in theme "white" are not high enough.

The current workaround is to use one of the other themes.
